sql/sys_vars.cc

```diff
@@ -7655,6 +7656,25 @@ static Sys_var_ulonglong Sys_tf_sequence_table_max_upper_bound(
     GLOBAL_VAR(tf_sequence_table_max_upper_bound), CMD_LINE(REQUIRED_ARG),
     VALID_RANGE(1024, ULLONG_MAX), DEFAULT(1048576), BLOCK_SIZE(1));
 
+static Sys_var_charptr Sys_var_buffered_error_log_filename(
```

Comment:
Also put them in alphabetical sort position? :thinking:

Reply:
That isn't an alphabetical list, entries before this are also in a random order.